__tests__/components/errorPage.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ErrorPage from '../../src/components/ErrorPage.jsx';
+
+describe('ErrorPage Component', () => {
+  it('should have an ID of error', () => {
+    const errorPage = mount(<ErrorPage />);
```

Comment:
You probably want to use shallow here instead of mount. as there are no lifeCycle method being tested here..